src/connectors/hierarchical-menu/__tests__/connectHierarchicalMenu-test.js

```diff
@@ -6,6 +6,33 @@ import { warning } from '../../../lib/utils';
 import connectHierarchicalMenu from '../connectHierarchicalMenu';
 
 describe('connectHierarchicalMenu', () => {
+  // @TODO: once we've migrate away from `getConfiguration` update
+  // the function and use it at least for the lifecycle.
```

Comment:
you mean that you kept it as a comment, because it was good, but right now unused?

Reply:
Yes we almost always perform the same setup.